e2e/tests/api-driven/src/permissions/helpers.ts

```diff
@@ -24,26 +28,27 @@ export const cleanup = async () => {
 };
 
 export const setup = async () => {
+  const user1Id = await createUser({ email: "e2e-user-1@opensystemslab.io" });
```

Comment:
nit: I think helen & mel previously reported an annoying issue where e2e test emails using the @opensystemslab.io domain were causing bounced errors in the enquiries inbox? can we use a harmless @example.com or something instead?